backend/src/test/java/io/debezium/configserver/LifecycleResourceIT.java

```diff
@@ -176,4 +177,36 @@ public void testRestartPostgresConnectorTaskWithInvalidTaskNumber() {
                 .body("message", is("Unknown task: " + connectorName + "-" + invalidTaskNumber));
     }
 
+    @Test
+    public void testGetPostgresConnectorConfigNotFound() {
```

Comment:
Tests for `ConnectorResource` methods are supposed to be in `ConnectorResourceIT`.
Lifecycle is more related to start, stop, delete of tasks. ConnectorResource is the right place, but could you move the test to the matching ConnectorResourceIT?

Reply:
yeah after thinking about it, realized it did not really fit the purpose of the other LifecycleResourceITs.  I will move them

Reply:
The PR has been updated